### ISSUE TITLE
unable to create file '/run/user/1000/dconf/user': Permission denied

### ISSUE THREAD

**Issue by eyalroz** [linuxmint/xreader#213]
I'm using:

 * xreader 1.6.2
 * Mint 18.3
 * x86_64 machine

I occasionally run `xdg-open my_file.pdf` from a terminal window, in which get several copies of the message:
```
    (xreader:11626): dconf-CRITICAL **: unable to create file '/run/user/1000/dconf/user': Permission denied.  dconf will not work properly.
```
This shouldn't happen... 

I'm not sure this is really an app-specific issue, as it pops up here and there with other apps and other distros. But I can't say.

**Comment by darran-kelinske-fivestars** [comment]
Did you find a resolution to this?

**Comment by eyalroz** [comment]
Not really; but I don't use Mint right now - I've switched to Devuan because Mint insists on using systemd.

**Comment by darran-kelinske-fivestars** [comment]
I fixed this with `sudo chown -R $USER:$USER /run/user`

**Comment by eyalroz** [comment]
1. That's a temporary fix. I had used it, but somehow the problem returned.
2. If you're not root, you can't do that.

**Comment by darran-kelinske-fivestars** [comment]
The problem returned for me too. did you find a permanent fix?
